presto-main/src/test/java/com/facebook/presto/operator/TestHashJoinOperator.java

```diff
@@ -25,6 +25,8 @@
 import com.facebook.presto.spi.Page;
 import com.facebook.presto.spi.block.Block;
 import com.facebook.presto.spi.type.Type;
+import com.facebook.presto.spiller.SingleStreamSpiller;
+import com.facebook.presto.spiller.SpillerFactory;
```

Comment:
They belong here. I have renamed commit to `Pass spiller parameters to ParallelHashBuildOperator`